apps/hermes/server/src/api/types.rs

```diff
@@ -271,9 +273,28 @@ impl From<PriceFeedUpdate> for ParsedPriceUpdate {
     }
 }
 
+#[derive(Debug, PartialEq, serde::Serialize, serde::Deserialize, Clone)]
+pub struct ParsedPublisherStakeCapsUpdate {
+    pub publisher_stake_caps: Vec<ParsedPublisherStakeCap>,
+}
+
+#[derive(Debug, PartialEq, serde::Serialize, serde::Deserialize, Clone)]
+pub struct ParsedPublisherStakeCap {
+    #[serde(with = "pyth_sdk::utils::as_string")]
+    pub publisher: Pubkey,
+    pub cap:       u64,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct GetPublisherStakeCapsUpdateDataResponse {
+    pub binary: BinaryUpdate,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub parsed: Option<Vec<ParsedPublisherStakeCapsUpdate>>,
+}
```

Comment:
This is only used in a single api right? please move it there.

Reply:
done